src/benchmarklib/benchmark_utils.hpp

```diff
@@ -37,16 +34,22 @@ using TableSegmentEncodingMapping =
  */
 std::ostream& get_out_stream(const bool verbose);
 
-struct QueryBenchmarkResult {
+struct QueryBenchmarkResult : public Noncopyable {
   QueryBenchmarkResult() { iteration_durations.reserve(1'000'000); }
+
+  QueryBenchmarkResult(QueryBenchmarkResult&& other) noexcept {
+    num_iterations.store(other.num_iterations);
+    duration = std::move(other.duration);
```

Comment:
move?

Reply:
Well, it's a move constructor. If `move` doesn't do anything for this member, so be it. At least we're future-safe if someone decides to change the type of duration.